wandb/sdk/artifacts/artifact.py

```diff
@@ -2297,6 +2297,12 @@ def _ttl_duration_seconds_from_gql(
         return None
 
 
+def is_require_core() -> bool:
+    if env.is_require_core():
+        return get_core_path() != ""
```

Comment:
It can't return an empty string, right? It could raise an error though. It will also log to the terminal whenever it is called; is that desired here?

Reply:
what do you mean log to the terminal?

Reply:
yeah, i need to change the return type i guess